lib/miq_expression.rb

```diff
@@ -257,9 +257,9 @@ def self._to_ruby(exp, context_type, tz)
       clause = "<find><search>" + _to_ruby(op_args["search"], context_type, tz) + "</search>" \
                "<check mode=#{mode}>" + _to_ruby(op_args[check], context_type, tz) + "</check></find>"
     when "key exists"
-      clause = operands2rubyvalue(operator, op_args, context_type)
+      clause, = operands2rubyvalue(operator, op_args, context_type)
```

Comment:
I am a little confused why the change in handling of the `operands2rubyvalue` values.

not quite sure the implication of having `clause` only be the first attribute - I don't think the values in that attribute changed

Reply:
yea revisiting this again, I still have no idea why you introduced this change. Please share